### ISSUE TITLE
Build fail related to ngfactory

### ISSUE THREAD

**Issue by jbw91** [driftyco/ionic-app-scripts#311]
#### Short description of the problem:

When I run `ionic run android --device` or `ionic build android` on [my project](https://github.com/jbw91/scripture-golf), then during the build I get the following console output and it fails:

```shell
> ionic-app-scripts build

[21:50:33]  ionic-app-scripts 0.0.39 
[21:50:33]  build prod started ... 
[21:50:33]  clean started ... 
[21:50:33]  clean finished in 4 ms 
[21:50:33]  copy started ... 
[21:50:33]  ngc started ... 
[21:50:33]  copy finished in 40 ms 
[21:50:33]  lint started ... 
[21:50:35]  lint finished in 1.62 s 
[21:50:43]  Error: Error at 
            /Users/username/dev/ionic/scripture-golf/.tmp/pages/game/game-gameplay/game-gameplay.ngfactory.ts:2233:38
[21:50:43]  Module ''*'' has no exported member 'Wrapper_NumberValueAccessor'. 
[21:50:43]  ngc failed 
[21:50:43]  ionic-app-script task: "build" 
[21:50:43]  Error: Error 

npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/username/.nvm/versions/node/v6.7.0/bin/node" "/Users/username/.nvm/versions/node/v6.7.0/bin/npm" "run" "build" "--color"
npm ERR! node v6.7.0
npm ERR! npm  v3.10.3
npm ERR! code ELIFECYCLE
npm ERR! scripture-golf@ build: `ionic-app-scripts build`
npm ERR! Exit status 1
```

#### What behavior are you expecting?

I'm expecting it to build properly so I can run it on my device.

**Steps to reproduce:**
1. Clone my repo [here](https://github.com/jbw91/scripture-golf)
2. Run `npm install`
3. Run `ionic build android`
4. Observe the error.

**Which @ionic/app-scripts version are you using?**

Version 0.0.39

**Other information:**

Output of `ionic info` command:

```shell
Your system information:

Cordova CLI: 6.3.1
Gulp version:  CLI version 1.2.2
Gulp local:   Local version 3.9.1
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.1
Ionic App Lib Version: 2.1.1
Ionic App Scripts Version: 0.0.39
ios-deploy version: 1.8.6 
ios-sim version: 5.0.8 
OS: Mac OS X Sierra
Node Version: v6.7.0
Xcode version: Xcode 8.0 Build version 8A218a 
```

**Comment by Atzie** [comment]
Seems to be a problem with building angular forms. No ngfactory file is being generated for `number_value_accessor.` Expected something similar to `default_value_accessor.ngfactory.ts`. Don't have a solution for this though.

**Comment by niconaso** [comment]
Same problem to me. When I try to compile my app I get this error:

[21:49:48]  Error: Error at 
            /Users/niconaso/desarrollo/workspace/runrun-3-mobile/.tmp/pages/comments/comments.ngfactory.ts:221:35 
[21:49:48]  Module ''*'' has no exported member 'Wrapper_ElasticDirective'. 
[21:49:48]  Error at 
            /Users/niconaso/desarrollo/workspace/runrun-3-mobile/.tmp/pages/new-post/new-post.ngfactory.ts:193:35 
[21:49:48]  Module ''*'' has no exported member 'Wrapper_ElasticDirective'. 
[21:49:48]  ngc failed 
[21:49:48]  ionic-app-script task: "build" 
[21:49:48]  Error: Error

**Comment by jbw91** [comment]
Further details: Apparently according to angular/angular#12428 this has to do with using an input element of `type="number"`, which is what I'm using in this case. This issue is in Angular 2.1.1. Anyway it appears that it has to do with Angular's AOT compilation as opposed to Ionic's build scripts.

**Comment by jbw91** [comment]
It appears to be fixed in Angular 2.1.2.

**Comment by jgw96** [comment]
Hello all! We just pushed a commit to the framework to update to the latest angular release `2.2.1` which fixes this issue for the next release. Thanks!
